.circleci/config.yml

```diff
@@ -223,23 +223,23 @@ jobs:
           name: Run build integration tests
           environment:
             - OKTETO_URL: https://staging.okteto.dev/
-            - OKTETO_PATH: 'C:\Users\circleci\project\artifacts\bin\okteto.exe'
+            - OKTETO_PATH: 'C:\Users\circleci.PACKER-64370BA5\project\artifacts\bin\okteto.exe'
             - OKTETO_SKIP_CLEANUP: 'true'
             - OKTETO_APPS_SUBDOMAIN: staging.okteto.net
           command: go test github.com/okteto/okteto/integration/build -tags="integration" --count=1 -v -timeout 10m
       - run:
           name: Run deploy integration tests
           environment:
             - OKTETO_URL: https://staging.okteto.dev/
-            - OKTETO_PATH: 'C:\Users\circleci\project\artifacts\bin\okteto.exe'
+            - OKTETO_PATH: 'C:\Users\circleci.PACKER-64370BA5\project\artifacts\bin\okteto.exe'
```

Comment:
where did you get this value from? (`circleci.PACKER-64370BA5`)

Reply:
From the `$HOME` variable. This orb uses this `$HOME` and I couldn't use `$HOME` substitution in the `OKTETO_PATH` (see [fix: circleci commit](https://github.com/okteto/okteto/pull/3779/commits/0819b3762d0b4e2890b116759d05d3fe27ca6a04))